### ISSUE TITLE
Export: OpenAIRE export format fails silently on more than one productionPlace (made multiple in recent pr)

### ISSUE THREAD

**Issue by kcondon** [IQSS/dataverse#9546]
This was discovered when testing DDI export which also saw this problem.
Recently the productionPlace field was made a multiple value field and that broke some export formats that assume only a single value. This issue is about OpenAire format that fails to generate an export file but reports no error.

For testing I used a manually created json file and used the api but simply adding two productionPlace fields should do it too.
[manual_all_fields.json.txt](https://github.com/IQSS/dataverse/files/11290528/manual_all_fields.json.txt)

curl -H X-Dataverse-key:xyz -X POST "http://localhost:8080/api/dataverses/root/datasets" --upload-file manual_all_fields.json -H 'Content-type:application/json'

**Comment by landreev** [comment]
Yes, I do believe that this is the same issue as what I found in the DDI exporter. I.e., another thing that needs to be changed after #9254. 

I'm seeing this line in OpenAireExportUtil.java:1334: 
`String geoLocationPlace = dto2Primitive(datasetVersionDTO, DatasetFieldConstant.productionPlace);`

In the DDI exporter I had to replace
`writeFullElement(xmlw, "prodPlac", dto2Primitive(version, DatasetFieldConstant.productionPlace));`
with
`writeFullElementList(xmlw, "prodPlac", dto2PrimitiveList(version, DatasetFieldConstant.productionPlace));`
But I'd like somebody who knows more about OpenAire to look into this.

**Comment by landreev** [comment]
I was briefly considering sneaking a fix for this into another OpenAire-related PR in the works. But I decided against it. Rather than making a quick fix, let's maybe invest a day of work into this and also add an integration test and maybe see if some code duplicated between this and the DDI export utils could be un-duplicated(?).

**Comment by pdurbin** [comment]
I'm able to reproduce this error on develop (d8480ff5d3). This new PR is supposed to fix it:

- #11194

I didn't commit the changes below. I added a second value to productionPlace...

```
% git diff
diff --git a/scripts/api/data/dataset-create-new-all-default-fields.json b/scripts/api/data/dataset-create-new-all-default-fields.json
index 151c4732ad..9c45849712 100644
--- a/scripts/api/data/dataset-create-new-all-default-fields.json
+++ b/scripts/api/data/dataset-create-new-all-default-fields.json
@@ -486,7 +486,7 @@
             "typeName": "productionPlace",
             "multiple": true,
             "typeClass": "primitive",
-            "value": ["ProductionPlace"]
+            "value": ["ProductionPlace", "another"]
           },
           {
             "typeName": "contributor",
diff --git a/src/test/java/edu/harvard/iq/dataverse/api/MetadataBlocksIT.java b/src/test/java/edu/harvard/iq/dataverse/api/MetadataBlocksIT.java
index 316ac579de..9a17f99664 100644
--- a/src/test/java/edu/harvard/iq/dataverse/api/MetadataBlocksIT.java
+++ b/src/test/java/edu/harvard/iq/dataverse/api/MetadataBlocksIT.java
@@ -1,6 +1,7 @@
 package edu.harvard.iq.dataverse.api;
 
 import io.restassured.RestAssured;
+import io.restassured.path.json.JsonPath;
 
 import io.restassured.response.Response;
 import org.hamcrest.CoreMatchers;
@@ -96,7 +97,7 @@ public class MetadataBlocksIT {
         assumeFalse(dataverseAlias == null || dataverseAlias.isBlank());
 
         // when
-        String pathToJsonFile = "scripts/api/data/dataset-create-new-all-default-fields.json";
+        String pathToJsonFile = "scripts/api/data/dataset-create-new-all-default-fields.json"; // what if productionPlace is multiple?
         Response createDataset = UtilIT.createDatasetViaNativeApi(dataverseAlias, pathToJsonFile, apiToken);
 
         // then
@@ -105,6 +106,15 @@ public class MetadataBlocksIT {
             ", response=" + createDataset.prettyPrint());
         createDataset.then().assertThat()
             .body("status", CoreMatchers.equalTo("OK"));
+
+        int datasetId = JsonPath.from(createDataset.body().asString()).getInt("data.id");
+        String datasetPid = JsonPath.from(createDataset.body().asString()).getString("data.persistentId");
+
+        UtilIT.publishDataverseViaNativeApi(dataverseAlias, apiToken).then().assertThat().statusCode(OK.getStatusCode());
+        UtilIT.publishDatasetViaNativeApi(datasetId, "major", apiToken).then().assertThat().statusCode(OK.getStatusCode());
+
+        Response openAireExport = UtilIT.exportDataset(datasetPid, "oai_datacite");
+        openAireExport.prettyPrint();
     }
 
 }
```

... and when I run the test above with `mvn test -Dtest=MetadataBlocksIT#testDatasetWithAllDefaultMetadata` I get the following error:

`[Fatal Error] :1:3997: XML document structures must start and end within the same entity.`
`<resource xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://datacite.org/schema/kernel-4" xsi:schemaLocation="http://datacite.org/schema/kernel-4 http://schema.datacite.org/meta/kernel-4.1/metadata.xsd"><identifier identifierType="DOI">10.5072/FK2/XNPSP2</identifier><creators><creator><creatorName nameType="Personal">LastAuthor1, FirstAuthor1</creatorName><givenName>FirstAuthor1</givenName><familyName>LastAuthor1</familyName><nameIdentifier nameIdentifierScheme="ORCID">AuthorIdentifier1</nameIdentifier><affiliation>AuthorAffiliation1</affiliation></creator><creator><creatorName nameType="Personal">LastAuthor2, FirstAuthor2</creatorName><givenName>FirstAuthor2</givenName><familyName>LastAuthor2</familyName><nameIdentifier nameIdentifierScheme="ISNI">AuthorIdentifier2</nameIdentifier><affiliation>AuthorAffiliation2</affiliation></creator></creators><titles><title>Replication Data for: Title</title><title titleType="Subtitle">Subtitle</title><title titleType="AlternativeTitle">Alternative Title</title></titles><publisher>Root</publisher><publicationYear>2025</publicationYear><subjects><subject>Agricultural Sciences</subject><subject>Business and Management</subject><subject>Engineering</subject><subject>Law</subject><subject schemeURI="http://KeywordVocabularyURL1.org" valueURI="http://keywordTermURI1.org" subjectScheme="KeywordVocabulary1">KeywordTerm1</subject><subject schemeURI="http://KeywordVocabularyURL2.org" valueURI="http://keywordTermURI2.org" subjectScheme="KeywordVocabulary2">KeywordTerm2</subject><subject schemeURI="https://TopicClassificationURL1.com" subjectScheme="Topic Classification Vocab1">Topic Classification Term1</subject><subject schemeURI="https://TopicClassificationURL2.com" subjectScheme="Topic Classification Vocab2">Topic Classification Term2</subject></subjects><contributors><contributor contributorType="ContactPerson"><contributorName nameType="Organizational">LastContact1, FirstContact1</contributorName><affiliation>ContactAffiliation1</affiliation></contributor><contributor contributorType="ContactPerson"><contributorName nameType="Organizational">LastContact2, FirstContact2</contributorName><affiliation>ContactAffiliation2</affiliation></contributor><contributor contributorType="Producer"><contributorName nameType="Personal">LastProducer1, FirstProducer1</contributorName><givenName>FirstProducer1</givenName><familyName>LastProducer1</familyName><affiliation>ProducerAffiliation1</affiliation></contributor><contributor contributorType="Producer"><contributorName nameType="Personal">LastProducer2, FirstProducer2</contributorName><givenName>FirstProducer2</givenName><familyName>LastProducer2</familyName><affiliation>ProducerAffiliation2</affiliation></contributor><contributor contributorType="DataCollector"><contributorName nameType="Personal">LastContributor1, FirstContributor1</contributorName><givenName>FirstContributor1</givenName><familyName>LastContributor1</familyName></contributor><contributor contributorType="DataCurator"><contributorName nameType="Personal">LastContributor2, FirstContributor2</contributorName><givenName>FirstContributor2</givenName><familyName>LastContributor2</familyName></contributor><contributor contributorType="Distributor"><contributorName nameType="Personal">LastDistributor1, FirstDistributor1</contributorName><givenName>FirstDistributor1</givenName><familyName>LastDistributor1</familyName><affiliation>DistributorAffiliation1</affiliation></contributor><contributor contributorType="Distributor"><contributorName nameType="Personal">LastDistributor2, FirstDistributor2</contributorName><givenName>FirstDistributor2</givenName><familyName>LastDistributor2</familyName><affiliation>DistributorAffiliation2</affiliation></contributor></contributors><dates><date dateType="Issued">1004-01-01</date><date dateType="Created">1003-01-01</date><date dateType="Submitted">1002-01-01</date><date dateType="Updated">2025-01-31</date><date dateType="Collected">1006-01-01`

**Comment by pdurbin** [comment]
I stashed my changes, jumped on this PR...

- #11194

... and applied them there. The bug seems to be fixed so I marked that PR as closing this issue. I'm seeing this in the output, which looks good (above I had added "another"):

    <geoLocation>
      <geoLocationPlace>ProductionPlace</geoLocationPlace>
    </geoLocation>
    <geoLocation>
      <geoLocationPlace>another</geoLocationPlace>
    </geoLocation>
    <geoLocation>
